package.json

```diff
@@ -52,5 +54,10 @@
     "test": "./node_modules/karma/bin/karma start --reporters spec,coverage",
     "lint": "fedx-scripts eslint --ext .jsx,.js openassessment/xblock/static/js/src/",
     "lint-fix": "fedx-scripts eslint --ext .jsx,.js openassessment/xblock/static/js/src/ --fix"
+  },
```

Comment:
does this override belong here, in the `openedx` repo?

Reply:
Because this fix apply to older version of frontend-build. I don't mind to just override it instead going through the whole deployment.